### PR TITLE
Change the number of points workers can give/earn 

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
Relates #733 #735

### CODE REVIEW

File: client/src/Components/Common/Navbar/Menu.js [hunk 1]
```diff
@@ -126,7 +130,7 @@ export default class Menu extends PureComponent {
             {isWorker && (
               <MenuItem to={MY_REVIEWS_URL} onClick={toggleMenu}>
                 <MenuIcon icon="starComment" height="19" width="19" />
-                Reviews you've given
+                Reviews you&apos;ve given
```
Comment (thejoefriel) [inline client/src/Components/Common/Navbar/Menu.js]: Out of curiosity, is this a better way to write an apostrophe in the code?

Comment (RamyAlshurafa) [reply client/src/Components/Common/Navbar/Menu.js]: I'm not sure but that was a warning from the eslint and they recommend to use this way

File: client/src/Components/Pages/Profile/Profile.style.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import {
   organizations,
   colors,
   shadows,
   borders,
-  breakpoints
-} from "./../../../theme";
+  breakpoints,
+} from "../../../theme";
 import { ReactComponent as ReplyIcon } from "../../../assets/reply-icon.svg";
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/Components/Pages/Profile/Profile.style.js]: that wasn't my code and i didn't change anything here, just fixed the eslint warnings, 
anyway I'm fixing them

[the remark this answers is not in the document]
Comment (thejoefriel) [reply client/src/Components/Pages/Profile/Profile.style.js]: Yeah I think this is most likely a legacy thing where ppl didn't always use the Icon component on previous sprints

**Pull request merged**